src/views/Inventory.vue

```diff
@@ -33,6 +33,24 @@
               </ion-label>
             </ion-item>
           </ion-card>
+
+          <ion-card>
+            <ion-card-header>
+              <ion-card-title>{{ $t("More jobs") }}</ion-card-title>
+            </ion-card-header>
+            <ion-item button @click="viewJobConfiguration('HARD_SYNC', 'Hard sync', getJobStatus(this.jobEnums['HARD_SYNC']))" detail>
```

Comment:
since these items will be populated dynamically without text in between, we should use an ion-list here

Reply:
Ok sir